server/controllers/userController.js

```diff
@@ -0,0 +1,77 @@
+import userModel from '../models/users';
+import encypt from '../helpers/encrypt';
+
+const { encryptPassword, decryptPassword, generateToken } = encypt;
+
+/**
+ *
+ *
+ * @export
+ * @class UsersController
+ */
+export default class UsersController {
+  /**
+   * Signup middleware - Create User Account
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {object} Object containing token to the user
+   * @memberof UsersController
+   */
+  static async signUp(req, res, next) {
+    try {
+      const data = req.body;
+      const { password } = data;
+      data.password = await encryptPassword(password);
+      const user = await userModel.create(data);
+      const token = await generateToken(user);
+
+      return res.status(201).json({
+        status: 'success',
+        data: { token }
+      });
+    } catch (err) {
+      return next(err);
+    }
+  }
+
+  /**
+   *
+   * Login Middleware - Logs user into the application
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   * @returns {object} Object containing token to the user
+   * @memberof UsersController
+   */
+  static async login(req, res, next) {
+    try {
+      const data = req.body;
+      const { userLogin, password } = data;
+      // Login with username or email address
+      const user = await userModel.findUserInput(userLogin);
+      if (!user) {
+        return res.status(404).json({
```

Comment:
API response should be extracted into a response file.

Reply:
@akpante3 I was assigned the task to create a response module and there's an open PR that implements it in #16

Reply:
Yes, this was implemented by @Xwebyna .. I noticed some stories on the PT has to be done together with some other stories.